lib/pos_mempool.go

```diff
@@ -118,6 +121,14 @@ type PosMempool struct {
 	// PosMempool only needs read-access to the block view. It isn't necessary to copy the block view before passing it
 	// to the mempool.
 	readOnlyLatestBlockView *UtxoView
+	// augmentedLatestBlockView is a copy of the latest block view with all the transactions in the mempool applied to
+	// it. This allows the backend to display the current state of the blockchain including the mempool.
+	// The augmentedLatestBlockView is updated every 100 milliseconds to reflect the latest state of the mempool.
+	augmentedLatestBlockView *UtxoView
```

Comment:
I'd rename this to readOnlyAugmentedLatestBlockView since we actually never modify it. Instead, we just always throw away the old one and let it get garbage-collected.

Reply:
Update: Resolved this myself.